### PR TITLE
chore: update inaccurate strings and formats

### PR DESCRIPTION
**Pull request by HuihuiWu-Microsoft**
ADO:
https://msazure.visualstudio.com/Microsoft%20Teams%20Extensibility/_workitems/edit/17000058
https://msazure.visualstudio.com/Microsoft%20Teams%20Extensibility/_workitems/edit/17000101
E2E TEST: NA

### CODE REVIEW

File: packages/fx-core/resource/package.nls.json [hunk 1]
```diff
@@ -90,11 +90,11 @@
   "core.progress.scaffoldBot": "Scaffolding Bot",
   "core.progress.scaffoldBot.detail": "Retrieving templates.",
   "core.progress.scaffoldApi": "Scaffolding Api",
-  "core.progress.scaffoldApi.detail": "Scaffolding Function Api project.",
+  "core.progress.scaffoldApi.detail": "Scaffolding Functions Api project.",
```
[the remark this answers is not in the document]
Comment (HuihuiWu-Microsoft) [reply packages/fx-core/resource/package.nls.json]: Updated Api/api -> API. 
For 'Function Api' here, I confirmed with @hund030 and it should be 'Functions project'.

**Review by hund030 — APPROVED** [review]

File: packages/fx-core/resource/package.nls.json [hunk 2]
```diff
@@ -277,7 +277,7 @@
   "plugins.appstudio.adminApprovalTip": "Successfully created %s app package file at %s. Send this to your administrator for approval.",
   "plugins.appstudio.adminPortal": "Go to admin portal",
   "plugins.appstudio.publishSucceedNotice": "[%s] successfully published to the admin portal. Once approved, your app will be available for your organization. [Learn more](%s)",
-  "plugins.appstudio.publishSucceedNotice.cli": "[%s] successfully published to the admin portal(%s). Once approved, your app will be available for your organization. Learn more from %s.",
+  "plugins.appstudio.publishSucceedNotice.cli": "[%s] successfully published to the admin portal (%s). Once approved, your app will be available for your organization. Learn more from %s.",
```
Comment (sffamily) [inline packages/fx-core/resource/package.nls.json:280]: before 'successfully' add 'has been'

Capitalize admin portal:  "Admin Portal"

File: packages/fx-core/resource/package.nls.json [hunk 3]
```diff
@@ -924,7 +924,7 @@
   "core.addSso.onlySupportAzure": "Add SSO is only supported for project hosted in Azure",
   "core.addSso.onlyMeNotSupport": "Add SSO is not supported on Message Extension only projects",
   "core.addSso.needCapability": "Add SSO is only supported for project with Tab, Bot and/or Message Extension. You need to run 'add component' first",
-  "core.addSso.functionNotSupport": "Add SSO is not supported for Bot hosted on Azure Function",
+  "core.addSso.functionNotSupport": "Add SSO is not supported for Bot hosted on Azure Functions",
```
Comment (sffamily) [inline packages/fx-core/resource/package.nls.json:927]: quote the "Add SSO"

File: packages/fx-core/resource/package.nls.json [hunk 4]
```diff
@@ -1050,8 +1050,8 @@
   "depChecker.needInstallFuncCoreTool": "You must have @NameVersion installed to debug your local functions.",
   "depChecker.failToInstallFuncCoreTool": "Failed to install @NameVersion. Install @NameVersion manually.",
   "depChecker.failToValidateFuncCoreTool": "Failed to validate @NameVersion after installation.",
-  "depChecker.globalFuncNodeNotMatched": "Your own Node.js(@NodeVersion) is incompatible with your own Azure Functions Core Tools(@FuncVersion). Please make sure your own Azure Functions Core Tools works with your Node.js version. See (@link) to Azure Functions supported Node.js versions.",
-  "depChecker.portableFuncNodeNotMatched": "Your own Node.js(@NodeVersion) is incompatible with Teams Toolkit Azure Functions Core Tools(@FuncVersion). Go to @Link to install Node.js (@RecommendedVersion is recommended).",
+  "depChecker.globalFuncNodeNotMatched": "Your own Node.js (@NodeVersion) is incompatible with your own Azure Functions Core Tools (@FuncVersion). Please make sure your own Azure Functions Core Tools works with your Node.js version. See (@link) to Azure Functions supported Node.js versions.",
```
Comment (sffamily) [inline packages/fx-core/resource/package.nls.json:1053]: remove all "own" from sentences and the sentence next.
last sentence:     to ->  for

File: packages/fx-core/resource/package.nls.json (hunk 2 above)
Comment (HuihuiWu-Microsoft) [reply packages/fx-core/resource/package.nls.json]: Updated

File: packages/fx-core/resource/package.nls.json (hunk 3 above)
Comment (HuihuiWu-Microsoft) [reply packages/fx-core/resource/package.nls.json]: Updated

File: packages/fx-core/resource/package.nls.json (hunk 4 above)
Comment (HuihuiWu-Microsoft) [reply packages/fx-core/resource/package.nls.json]: Updated

**Review by sffamily — DISMISSED** [review]

**Review by sffamily — APPROVED** [review]

**Pull request merged**